composer.json

```
@@ -13,7 +13,7 @@
     "homepage": "http://github.com/apigee/apigee-client-php",
     "license": "Apache-2.0",
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.0",
```

Be more specific about actually supported (and tested) PHP versions.

See: https://github.com/apigee/apigee-edge-drupal/pull/816


```suggestion
        "php": "~8.0.0 || ~8.1.0",
```